build-system/compile/compile.js

```diff
@@ -266,7 +266,7 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
       assume_function_wrapper: true,
       // Transpile from ES6 to ES5.
       language_in: 'ECMASCRIPT6',
-      language_out: 'ECMASCRIPT5',
+      language_out: argv.esm ? 'NO_TRANSPILE' : 'ECMASCRIPT5',
```

Comment:
Nit: Clarify the comment above (Transpile from ES6 to ES5) since it's no longer accurate for all scenarios.